saleor/dashboard/emails.py

```diff
@@ -11,35 +13,70 @@
 from ..core.utils import build_absolute_uri
 
 
-def _send_set_password_email(pk, template_name):
-    user = User.objects.get(pk=pk)
-    uid = urlsafe_base64_encode(force_bytes(user.pk))
-    token = default_token_generator.make_token(user)
+def send_set_password_staff_email_with_url(redirect_url, staff):
```

Comment:
Do we want to have a separate function for customer and staff?
We could do it in single function:
```
def send_set_password_email_with_url(redirect_url, customer, staff=False):
    """Trigger sending a set password email for the given customer/staff."""
    template_type = 'staff' if staff else 'customer'
    template = f"dashboard/{template_type}/set_password"
    token = default_token_generator.make_token(customer)
    _send_set_user_password_email_with_url.delay(
        customer.email, redirect_url, token, template
    )
```
Similar think is with `send_set_password_customer_email_with_url`

Reply:
And the same think with `send_set_password_staff_email`
There are so many functions starting `send_set_password...` here. So maybe for better readability, we could create easily one function for customer and staff